.docker/jats-ingester/airflow.cfg

```diff
@@ -295,7 +298,8 @@ dag_default_view = tree
 
 # Default DAG orientation. Valid values are:
 # LR (Left->Right), TB (Top->Bottom), RL (Right->Left), BT (Bottom->Top)
-dag_orientation = LR
+# dag_orientation = LR
+dag_orientation = TB
```

Comment:
is this just a UI change?

Reply:
yes, as a DAG gains more tasks the current view becomes wider and harder to read. I saw this setting to view it vertically in the UI which should make longer DAGs easier to read.